README.md

```diff
@@ -1 +1,320 @@
+
+Also, there is another placeholder named `<BALLERINA_LAYER_ARN>`, which is the ARN of the Ballerina runtime layer. There will be a Ballerina runtime layer for each Ballerina release, thus, the specific `<BALLERINA_LAYER_ARN>` should be looked up from the table available at [https://ballerina.io/deployment/aws-lambda](https://ballerina.io/deployment/aws-lambda).
+
+With the above placeholder values at hand, we can start to deploy the function we have developed. This consists of running the `aws lambda create-function` and the `aws lambda update-function-configuration` commands to finish the deployment. Below contains sample build commands for deploying the `processImages` function.
+
+- Deploying to AWS Lambda
+
+```bash
+$ aws lambda create-function --function-name processImages --zip-file fileb://aws-ballerina-lambda-functions.zip --handler image-text-processor.processImages --runtime provided --role arn:aws:iam::908363916138:role/lambda-role --timeout 10 --memory-size 1024
+{
+    "TracingConfig": {
+        "Mode": "PassThrough"
+    }, 
+    "CodeSha256": "lwg8/90iroOzc5fFYU4Fp9r9CpMnsWgx2xOPekldHdA=", 
+    "FunctionName": "processImages", 
+    "CodeSize": 41462, 
+    "RevisionId": "ba962bb9-976e-47f1-9349-cc79319cc30d", 
+    "MemorySize": 1024, 
+    "FunctionArn": "arn:aws:lambda:us-west-2:908363916138:function:processImages", 
+    "Version": "$LATEST", 
+    "Role": "arn:aws:iam::908363916138:role/lambda-role", 
+    "Timeout": 10, 
+    "LastModified": "2019-02-02T00:28:02.658+0000", 
+    "Handler": "image-text-processor.processImages", 
+    "Runtime": "provided", 
+    "Description": ""
+}
+
+$ aws lambda update-function-configuration --function-name processImages --layers arn:aws:lambda:us-west-2:908363916138:layer:ballerina-0_990_3-runtime:19
+{
+    "Layers": [
+        {
+            "CodeSize": 175675006, 
+            "Arn": "arn:aws:lambda:us-west-2:908363916138:layer:ballerina-0_990_3-runtime:19"
+        }
+    ], 
+    "FunctionName": "processImages", 
+    "LastModified": "2019-02-02T00:30:55.803+0000", 
+    "RevisionId": "7637cae8-703b-489a-87af-ecdae62b1816", 
+    "MemorySize": 1024, 
+    "Version": "$LATEST", 
+    "Role": "arn:aws:iam::908363916138:role/lambda-role", 
+    "Timeout": 10, 
+    "Runtime": "provided", 
+    "TracingConfig": {
+        "Mode": "PassThrough"
+    }, 
+    "CodeSha256": "lwg8/90iroOzc5fFYU4Fp9r9CpMnsWgx2xOPekldHdA=", 
+    "Description": "", 
+    "CodeSize": 41462, 
+    "FunctionArn": "arn:aws:lambda:us-west-2:908363916138:function:processImages", 
+    "Handler": "image-text-processor.processImages"
+}
+```
+
+- Updating Connector Credential Environment Variables
```

Comment:
I think it is better if we can make this a sub-title.

Reply:
Yes, true, thanks for the suggestion.